### DISCUSSION TITLE
Welcome new committer: Yan Zhao

### DISCUSSION THREAD

**Discussion by fengjiachun** [sofastack/sofa-jraft#691] [Announcements]
Hi Community,

On behalf of the SOFAJRaft PMC, I am pleased to announce that [Yan Zhao](https://github.com/horizonzy)  a.k.a horizonzy , has been voted in as SOFAJRaft committer.

Please join me to say congratulations to him!

Best Regards,
Block

**Comment by killme2008** [comment]
Welcome !

**Comment by khotyn** [comment]
Congrats！

**Comment by hzh0425** [comment]
congratulations!!!!!
